styles/attendees.module.css

```diff
@@ -0,0 +1,181 @@
+@import url('https://fonts.googleapis.com/css2?family=Martel+Sans:wght@400;700;800&display=swap');
```

Comment:
Do not use any new font, kindly use the font variable available in the global CSS file.